modules/apps/object/object-service/src/main/resources/resource-actions/default.xml

```diff
@@ -44,6 +44,26 @@
 			</guest-unsupported>
 		</permissions>
 	</model-resource>
+	<model-resource>
+		<model-name>com.liferay.object.model.ObjectEntryFolder</model-name>
+		<portlet-ref>
+			<portlet-name>com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet</portlet-name>
+		</portlet-ref>
+		<weight>1</weight>
+		<permissions>
+			<supports>
+				<action-key>ADD_FOLDER</action-key>
+				<action-key>DELETE</action-key>
+				<action-key>UPDATE</action-key>
+				<action-key>VIEW</action-key>
+			</supports>
+			<guest-unsupported>
+				<action-key>ADD_FOLDER</action-key>
+				<action-key>DELETE</action-key>
+				<action-key>UPDATE</action-key>
+			</guest-unsupported>
+		</permissions>
+	</model-resource>
```

Comment:
@gabrielwas please see @adolfopa's comment about permissions in the PR description ([here](https://github.com/liferay-bpm/liferay-portal/pull/3947#issue-2727108458)).